arktype.io/src/css/custom.css

```diff
@@ -63,3 +63,31 @@ header {
     overflow: hidden;
     background: #0067a5;
 }
+
+#featuresComponent {
```

Comment:
In general, we are only using CSS for docusaurus components we don't have direct control over. Otherwise, we have just been using inline styles or the relatively equivalent-looking `sx` prop on MUI components.